hypershift-operator/controllers/hostedcluster/internal/platform/openstack/openstack.go

```diff
@@ -272,6 +272,7 @@ func (a OpenStack) CAPIProviderDeploymentSpec(hcluster *hyperv1.HostedCluster, _
 				"--namespace=$(MY_NAMESPACE)",
 				"--leader-elect",
 				"--health-probe-bind-address=:8081",
+				"--zap-log-level=4",
```

Comment:
Should this always be set to 4 or to some configurable value?

Reply:
Ideally this would use the same log level as everything else. It's a bug in ORC that we can't do that. Possibly worth adding a comment (with a bug link?) explaining why we're hard coding 4 here, but this is an appropriate workaround for now IMHO.

Reply:
The logs in OpenStack-Resource-Controller (ORC) aren't yet well implemented and from my experience as a user, log level 4 (INFO) is what provides useful information for now for all resources that ORC handles.
Would you have a preference on how we can configure it?

Reply:
I'll add a link so we can keep track of it.

Reply:
Maybe you could add it as a constant here in an OpenStack section - hypershift/support/config/constants.go? That way a comment could be added why its 4 as Matthew suggested. WDYT?